### PR TITLE
feat(ui): StyleGuide (Box and Card)

### LINKED ISSUE #301: Feature/Style Guide - Box and Card Components
## The story:
As a developer/designer, I want to see a style guide for the box and card components.

---

## What we should do:
We should create a style guide for the box and card components.

---

## To be successful we need to:
_Check these off as they are completed:_
- [ ] Box style guide
- [ ] Card style guide

---

### PR DESCRIPTION
**Pull request by suzynakayama**
- box
- card

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues (PRs without this will not be accepted)
- Fixes #301

## Changes
- Added Box story
- Added Card story
- Added both to stories.ts

## PR Checklist
- [ ] Does your PR contain relevant tests that pass?
- [x] Is your code linted? `yarn lint` or `yarn lint:fix`
- [x] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: packages/ui/components/Box/Box.story.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Box
+
+## Properties
+There are three properties that can be used with the Box component: `padding`, `margin`, `shadow`. All three are optional.
+- Padding and Margin: it can be a `number` or a `Size` (from the theme). If not set, it will use the `md` size (1.6rem).
+
+- Shadow: it can be one of the theme's options (`main`, `success`, `error`). If not set, the box will have no shadows.
+
+```tsx
+<Box>...content...</Box>
```
Comment (tristanMatthias) [inline packages/ui/components/Box/Box.story.mdx:27]: Hey @suzynakayama! Good explanation of this stuff. Would you mind doing a case for each, using the formt from say the Button story? Thanks!

**Review by tristanMatthias — CHANGES_REQUESTED** [review]
Heyy @suzynakayama ! Thanks for this. Added a comment to expand upon the variations with some visual examples/headings etc. Let me know if it's unclear 😄

File: packages/ui/components/Box/Box.story.mdx (hunk 1 above)
Comment (suzynakayama) [reply packages/ui/components/Box/Box.story.mdx:27]: Just pushed a new version.

**Comment by suzynakayama** [comment]
> Heyy @suzynakayama ! Thanks for this. Added a comment to expand upon the variations with some visual examples/headings etc. Let me know if it's unclear 😄

Just pushed a new version! =p

**Review by tristanMatthias — APPROVED** [review]

**Pull request merged**